src/styles.scss

```diff
@@ -308,7 +308,7 @@ body {
     height: $view-container-height;
   }
 
-  planet-resources-viewer div * {
+  planet-resources-viewer div > video {

```

Comment:
Could we add this as part of the PlanetResourcesViewer styles instead of the main style.scss file?